pkg/config/config.go

```diff
@@ -282,6 +282,15 @@ type Release struct {
 	ExtraFiles   []ExtraFile `yaml:"extra_files,omitempty"`
 }
 
+// Milestone config used for VCS milestone.
+type Milestone struct {
+	GitHub       Repo   `yaml:",omitempty"`
+	GitLab       Repo   `yaml:",omitempty"`
+	Gitea        Repo   `yaml:",omitempty"`
```

Comment:
Generally the way authentication is done currently we don't support more than 1 provider at a time, so you could not e.g. close a milestone in GitHub and GitLab at the same time. Only a single token is expected at a time.

Relatedly I recently refactored similar granular logic in Homebrew https://github.com/goreleaser/goreleaser/pull/1649

So I think we should just reduce these to `Repo Repo `yaml:",omitempty"`

Reply:
True that... I think it makes sense to allow a single repo indeed.

Reply:
Updated to single `repo` configuration in e95effc3db84a45d4a067a428490945baf7057df 👍